test/server/configuration_impl_test.cc

```diff
@@ -303,6 +303,55 @@ TEST_F(ConfigurationImplTest, StatsSinkWithNoName) {
                             "Provided name for static registration lookup was empty.");
 }
 
+TEST_F(ConfigurationImplTest, StaticSecretRead) {
+  std::string json =
+      R"EOF(
+    {
+      "listeners" : [
+        {
+          "address": "tcp://127.0.0.1:1234",
+          "filters": []
+        }
+      ],
+      "cluster_manager": {
+        "clusters": []
+      },
+      "admin": {"access_log_path": "/dev/null", "address": "tcp://1.2.3.4:5678"}
+    }
+    )EOF";
+
+  envoy::config::bootstrap::v2::Bootstrap bootstrap = TestUtility::parseBootstrapFromJson(json);
+
+  auto secret_config = bootstrap.mutable_static_resources()->mutable_secrets()->Add();
+
+  std::string yaml =
+      R"EOF(
+  name: "abc.com"
```

Comment:
can we combine this static secret config with overall config defined in line 307?   I think it will be easier to read.

Reply:
Per offline discussion, I am going to keep it this way.